src/backend/tests/models/charge_types/test_felony_class_a.py

```diff
@@ -2,13 +2,37 @@
 
 from datetime import datetime, timedelta
 
+from expungeservice.models.charge_types.felony_class_a import FelonyClassA
+from expungeservice.models.disposition import Disposition
 from expungeservice.models.expungement_result import EligibilityStatus
 
 from tests.factories.charge_factory import ChargeFactory
-from expungeservice.models.disposition import Disposition
 
 
-class TestSingleChargeAcquittals(unittest.TestCase):
+class TestSingleChargeConvictionsFelonyClassA(unittest.TestCase):
```

Comment:
Seems like for Class A/B/C, all of these testing classes share the setup/create_recent_charge methods, should they be refactored into a generic `TestSingleChargeConvictionsFelony(unittest.TestCase)` base class, which then ``TestSingleChargeConvictionsFelonyClass{A/B/C}(`TestSingleChargeConvictionsFelony)` could inherent from?